experimental/sev_guest/src/guest.rs

```diff
@@ -47,11 +49,20 @@ pub struct GuestMessage {
 
 static_assertions::assert_eq_size!(GuestMessage, [u8; GUEST_MESSAGE_SIZE]);
 
+impl GuestMessage {
+    pub const fn new() -> Self {
+        GuestMessage {
+            header: GuestMessageHeader::new(),
+            payload: [0; 4000],
+        }
+    }
+}
```

Comment:
Should this be Default (and automatically derived)? Or does that not work because it's not const?

Reply:
Default::default is not const, and also Default is only implemented for fixed sized arrays up to 32 elements. Arrays larger than that is not supported when deriving Default.